docker-compose.yaml

```diff
@@ -47,22 +47,19 @@ services:
   summerjob-planner:
     container_name: summerjob-planner
     hostname: summerjob-planner
-    image: "ghcr.io/ladal1/summerjob-planner:latest"
```

Comment:
You can't remove the image, that would break pulling the image from the container repository here - we're not building it on the end machines, they need option for pulling image